modules/lib.lua

```diff
@@ -341,12 +344,19 @@ else
         local color = params.color or vec4(255, 255, 255, 255)
         local width = params.width or 1.0
         local height = params.height or 1.0
+        local disableDropShadow = params.disableDropShadow or false
+        local disableOutline = params.disableOutline or false
```

Comment:
Why does 2D use disable while 3D uses enable? 2D defaults these features to on if unset whereas 3D does not.

Reply:
> Why does 2D use disable while 3D uses enable? 2D defaults these features to on if unset whereas 3D does not.

The reason I did it like that is because before I changed stuff there was a outline and drop shadow on the 2d text by default and none on the 3d text by default. So I wanted to keep it like that.  

This is the original code for drawText2d as you can see by default there is a outline and drop shadow
```lua
    function qbx.drawText2d(params)
        local text = params.text
        local coords = params.coords
        local scale = params.scale or 0.35
        local font = params.font or 4
        local color = params.color or vec4(255, 255, 255, 255)
        local width = params.width or 1.0
        local height = params.height or 1.0
        SetTextScale(scale, scale)
        SetTextFont(font)
        SetTextColour(math.floor(color.r), math.floor(color.g), math.floor(color.b), math.floor(color.a))
        SetTextDropShadow()
        SetTextOutline()
        SetTextCentre(true)
        BeginTextCommandDisplayText('STRING')
        AddTextComponentSubstringPlayerName(text)
        EndTextCommandDisplayText(coords.x - width / 2, coords.y - height / 2 + 0.005)
    end
```

Reply:
Do you want me to change it? Or can I keep it the way I've done?

Reply:
@Manason am I allowed to mark this conversation as resolved?

Reply:
Yeah let's default it either way for both consistently.

Reply:
Not sure I totally understand, what you mena by that. Can you elaborate?

Reply:
Let's either default shadow to on and have the disable param, or default it to off and have the enable param. But I don't want them to have different params.

Reply:
Okay, what would you prefer?

Reply:
Probably default off with an optional enable param

Reply:
Okay, will change it quick. I also have a updated public documentation fork, where I will change the stuff on too. And make a pull request on the docs repository.